librecad/src/lib/engine/rs_ellipse.cpp

```diff
@@ -1364,8 +1416,39 @@ void RS_Ellipse::scale(const RS_Vector& center, const RS_Vector& factor) {
 // calculateBorders();
 
 }
+*/
+
+void RS_Ellipse::scale(const RS_Vector& center, const RS_Vector& factor) 
```

Comment:
scale(center, factor) means the scaling is centered the the parameter center, and it means the center is invariant before and after the scaling. All other points scales according to relative position to this scaling center.